README.md

```diff
@@ -261,6 +261,34 @@ and
 instance.sendCoin([address, amount], {from: account[1]});
 ```
 
+# Verifying the PGP signature
+
+Prepare, you need to install the npm [pkgsign](https://www.npmjs.com/package/pkgsign#installation) for verifying.
+
+First, get the version of tronbox dist.tarball
+
+```shell
```

Comment:
use `console` for command line demostration.
use `shell` or `sh` for plain command.

Reply:
OK